### ISSUE TITLE
Change dates from time-zone-aware timestamp to naive timestamp?

### ISSUE THREAD

**Issue by AmandaBirmingham** [jdereus/labman#329]
@wasade @tanaes @ElDeveloper please weigh in:

Currently the labman database stores time-zone aware dates, in order to be technically able to differentiate potentially different times (e.g., 2017-10-25 19:10:25 in New York is not the same time as 2017-10-25 19:10:25 in Beijing).  However, this incurs additional steps when unit-testing string outputs containing dates and times, as the strings differ based on the time zone in which the test is run and so have to be generated within the test rather than hard-coded (see an example below.)

@antgonza suggests changing the date fields in the database from timezone-aware to timezone-naive (timestamptz to timestamp), since labman is not anticipated to be run in multiple locations.  Shall we?

https://github.com/jdereus/labman/blob/ba272c7ab0e197c094e24a9630c64dc6f72da9ea/labman/db/tests/test_process.py#L1496-L1510

**Comment by antgonza** [comment]
As you guys (@wasade @tanaes @ElDeveloper) can imagine this is blocking finishing the solution for #301 so it will be great to hear from you. 

Now, I was working on the changes to make this happen - so we could have a better discussion point - and realized a new thing: [get_date_format](https://github.com/jdereus/labman/blob/master/labman/db/process.py#L88) is format as: `'%Y-%m-%d %H:%M'` but all the tests are like `'%Y-%m-%d %H:%M:%S'`, which has this not fun implication:
```
>>> from datetime import datetime
>>> datetime.strptime('2018-01-01 00:00:00', '%Y-%m-%d %H:%M:%S')
datetime.datetime(2018, 1, 1, 0, 0)
>>> datetime.strptime('2018-01-01 00:00:00', '%Y-%m-%d %H:%M')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/antoniog/miniconda3/envs/labman/lib/python3.5/_strptime.py", line 510, in _strptime_datetime
    tt, fraction = _strptime(data_string, format)
  File "/Users/antoniog/miniconda3/envs/labman/lib/python3.5/_strptime.py", line 346, in _strptime
    data_string[found.end():])
ValueError: unconverted data remains: :00
>>> datetime.strptime('2018-01-01 00:00', '%Y-%m-%d %H:%M:%S')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/antoniog/miniconda3/envs/labman/lib/python3.5/_strptime.py", line 510, in _strptime_datetime
    tt, fraction = _strptime(data_string, format)
  File "/Users/antoniog/miniconda3/envs/labman/lib/python3.5/_strptime.py", line 343, in _strptime
    (data_string, format))
ValueError: time data '2018-01-01 00:00' does not match format '%Y-%m-%d %H:%M:%S'
>>> datetime.strptime('2018-01-01 00:00', '%Y-%m-%d %H:%M')
datetime.datetime(2018, 1, 1, 0, 0)
```
In other words, this is currently working because the [tests are using their own formatting in a test by test case](https://github.com/jdereus/labman/blob/master/labman/db/tests/test_process.py#L48-L55) vs. only using get_date_format.
